plugins/CMakeLists.txt

```diff
@@ -55,10 +55,14 @@ foreach(plugin ${plugins})
     message(STATUS "Plugin ${plugin_name} provides modules: ${plugin_modules}")
     add_library(${plugin_name} ${rime_plugin_boilerplate_src} ${plugin_objs})
     target_link_libraries(${plugin_name} ${plugin_deps})
+    set_target_properties(${plugin_name}
+      PROPERTIES
+      LIBRARY_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/lib/${RIME_PLUGINS_DIR}
+      RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/lib/${RIME_PLUGINS_DIR})
```

Comment:
try this:
set `RUNTIME_OUTPUT_DIRECTORY`  to `${PROJECT_BINARY_DIR}/bin/${RIME_PLUGINS_DIR}`
do so also for `rime.dll`.

if it worked, we might be able to run `rime_console` from `build\bin` without copying `rime.dll` there.

Reply:
done.